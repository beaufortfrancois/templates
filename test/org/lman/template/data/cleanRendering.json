{
  "list1": [1, 2, 3],
  "list2": [4, 5, 6],
  "list3": [7, 8, 9],
  "list4": [10],
  "list5": [
    {"value": "a"},
    {"value": "b"},
    {"value": "c"}
  ]
}
