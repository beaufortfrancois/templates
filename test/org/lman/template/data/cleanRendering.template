Give me a one two three.
{{#list1}}
  {{@}}
{{/}}
  Give me a four five six.
  {{#list2}}
    {{@}}
  {{/}}

Give me a {{#list3}}{{@}}{{/}} (seven eight nine).
  Give me a {{#list4}}{{@}}{{/}} (TEN!).

Give me an abc.
{{#list5}}
  {{*@}}
  {{@.value}}
  {{value}}
{{/}}
  Give me an abc.
  {{#list5}}
    {{*@}}
    {{@.value}}
    {{value}}
  {{/}}

Should render blank lines between each of these:
{{#list1}}
  {{@}}

{{/}}
  {{#list1}}
    {{@}}

  {{/}}

Should render each on its own line:
{{#list1}}{{@}}{{/}}
  {{#list2}}{{@}}{{/}}

Nothing to see here.
{{#nothing1}}
  Seriously, nothing to see here.
{{/}}

Nothing to see here either.
{{#nothing2}}Foo{{/}}
  {{#nothing3}}Bar{{/}}

Done!
