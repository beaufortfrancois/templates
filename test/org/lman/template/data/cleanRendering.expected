Give me a one two three.
  1
  2
  3
  Give me a four five six.
    4
    5
    6

Give me a 789 (seven eight nine).
  Give me a 10 (TEN!).

Give me an abc.
  {"value":"a"}
  a
  a
  {"value":"b"}
  b
  b
  {"value":"c"}
  c
  c
  Give me an abc.
    {"value":"a"}
    a
    a
    {"value":"b"}
    b
    b
    {"value":"c"}
    c
    c

Should render blank lines between each of these:
  1

  2

  3

    1

    2

    3


Should render each inline:
123
  456

Nothing to see here.

Nothing to see here either.

  

Done!
